docs/src/windows.md

```diff
@@ -0,0 +1,106 @@
+# Install and contribute to Erdapfel on Windows
+
+## Windows 7 / 8
```

Comment:
As Win7 is old and not supported anymore, and Win8 is quasi non-existant, I would consider Windows 10 the default case and put it first. Especially since it doesn't require full Linux virtualization.

Reply:
Agreed: I'm commenting this section altogether. It won't be visible in the help page, but still present in the doc's source code just in case